src/adapters/pois_styles.js

```diff
@@ -0,0 +1,10 @@
+
+export const filteredPoisStyle = {
+  type: 'symbol',
+  layout: {
+    'icon-image': 'pin_with_dot',
+    'icon-allow-overlap': true,
```

Comment:
I am not sure it would make any difference here, but it would be more consistent to me to set [`icon-ignore-placement`](https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#layout-symbol-icon-ignore-placement) too:
```js
'icon-ignore-placement': true,
```

Reply:
Ah yes, thanks! More street names are visible this way.